kcqrs-client/src/main/kotlin/com/clouway/kcqrs/client/SyncEventPublisher.kt

```diff
@@ -0,0 +1,18 @@
+package com.clouway.kcqrs.client
+
+import com.clouway.kcqrs.core.*
+
+/**
+ * @author Miroslav Genov (miroslav.genov@clouway.com)
+ */
+class SyncEventPublisher(private val messageBus: MessageBus) : EventPublisher {
+    override fun publish(events: Iterable<EventWithPayload>) {
+        events.forEach {
+            try {
+                messageBus.handle(it)
+            } catch (ex: Exception) {
```

Comment:
I think it would be better the MessageBus to throw Exceptions that extend MessageBusException for instance. In my opinion it's bad practice to catch all exceptions.

Reply:
This means that you have to force client code to throw specific exception which doesn't sound good. Keep in mind that EventHandler is implemented by the client code.